### DISCUSSION TITLE
"Replace" relation list

### DISCUSSION THREAD

**Discussion by georgekrax** [prisma/prisma#10699] [Q&A] [answered]
Hello everyone,

Is there a way with Prisma to replace / set explicitly the relations list of a model field? Like override all relations and pass this specific list? Something like this is easy to implement with TypeORM, by:

```js
model.filters = newFilters;
await model.save()
```

Here is a related [question](https://stackoverflow.com/questions/69804623/prisma-set-with-explicit-many-to-many-relation) in stackoverflow.com

Thanks in advance

**Comment by chemicalkosek** [comment, marked as answer]
Yes, you can use `set`: https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#set
Is that not working for you?

**Reply by georgekrax** [reply]
At first, it was not working because I had explicitly set Many-to-Many relations. Now yes, with your help, I managed to solve it. Thank you!

**Reply by georgekrax** [reply]
@chemicalkosek Do you know also where I can find the "full" model types of my schema? The generated ones provided by Prisma do not list relations, and I would like to include them also in a model.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
